src/librustc/lint/context.rs

```diff
@@ -564,6 +564,11 @@ impl LintPassObject for EarlyLintPassObject {}
 
 impl LintPassObject for LateLintPassObject {}
 
+#[derive(Clone, Copy)]
+pub enum ItemId {
```

Comment:
This should be removed.

Reply:
Good call 👍; I was worried that without it we would need to perform many conversions, but it turned out really well.